pkgs/by-name/pl/plattenalbum/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  gobject-introspection,
+  wrapGAppsHook4,
+  desktop-file-utils,
+  libadwaita,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "plattenalbum";
+  version = "2.1.0";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "SoongNoonien";
+    repo = "plattenalbum";
+    rev = "v${version}";
+    hash = "sha256-vRBlShbNuPpL29huhzYSuUcMJmSLljO4nc6cSAp3NB4=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    gobject-introspection
+    wrapGAppsHook4
+    desktop-file-utils
+  ];
+
+  buildInputs = [ libadwaita ];
+
+  dependencies = with python3Packages; [
+    pygobject3
+    mpd2
+  ];
+
+  dontWrapGApps = true;
+
+  preFixup = ''
+    makeWrapperArgs+=(''${gappsWrapperArgs[@]})
+  '';
+
+  meta = {
+    description = "A client for the Music Player Daemon (originally named mpdevil)";
+    homepage = "https://github.com/SoongNoonien/plattenalbum";
```

You could add a changelog entry.

```suggestion
    homepage = "https://github.com/SoongNoonien/plattenalbum";
    changelog = "https://github.com/SoongNoonien/plattenalbum/releases/tag/v${version}";
```